modules/foonathan_memory/0.7.3/MODULE.bazel

```diff
@@ -0,0 +1,38 @@
+module(
+    name = "foonathan_memory",
+    version = "0.7.3",
+    compatibility_level = 1,
+)
+
+bazel_dep(
+    name = "rules_foreign_cc",
+    version = "0.10.1",
+)
+
+tools = use_extension("@rules_foreign_cc//foreign_cc:extensions.bzl", "tools")
+
+use_repo(
+    tools,
+    "cmake_3.23.2_toolchains",
+    "cmake_src",
+    "gnumake_src",
+    "meson_src",
+    "ninja_1.11.1_toolchains",
+    "ninja_build_src",
+    "pkgconfig_src",
+    "rules_foreign_cc_framework_toolchains",
+)
+
+register_toolchains(
```

Comment:
Doesn't rules_foreign_cc register these toolchains? I'm worried that this could have unexpected side effects for projects that use rules_foreign_cc themselves.

Reply:
@fmeum I can't fix the ci error with macos_arm64 and windows due to foreign_cc toolchain, so I remove register_toolchains and these platform from presubmit once.